config/Dockerfiles/fedoraci-runner/package-test/package-test.sh

```diff
@@ -187,6 +216,12 @@ for playbook in tests*.yml; do
         provision_with_retry
         ANSIBLE_STDOUT_CALLBACK=yaml timeout 4h ansible-playbook -v --inventory=pipeline_inventory.yaml $PYTHON_INTERPRETER \
             --tags ${TAG} ${playbook} $@ | tee ${TEST_ARTIFACTS}/${playbook}-run.txt
+
+        # if there are multiple playbooks, each new invocation would overwrite
+        # the previous results.yml. therefore we merge the new results
+        # with previous results here.
+        merge_results ${TEST_ARTIFACTS}/results.yml.prev ${TEST_ARTIFACTS}/results.yml
+        cp ${TEST_ARTIFACTS}/results.yml ${TEST_ARTIFACTS}/results.yml.prev
```

Comment:
With the use of commands like diff, sed, and cp... make sure the script explicitly sets a PATH variable.  I usually start with this for scripts:

PATH=/bin:/usr/bin

For portability.  If I need sbin commands, I expand the path more.

Reply:
Yeah, this is probably a good idea. Although I am not sure what will/could break if I do this (this is my first commit to this repository).

But it seems like there are some tests that ran on this pull requests, so hopefully it would blow up on me if I change it.

Reply:
CI is green :)